app/scripts/modules/core/src/navigation/urlBuilder.service.ts

```diff
@@ -352,14 +335,14 @@ export class UrlBuilderService {
     let result: string | boolean;
     if (!asgName) {
       result = false;
-    } else if (!asgName.match(UrlBuilderService.PUSH_VERSION)) {
+    } else if (!asgName.match(NamingService.VERSION_PATTERN)) {
```

Comment:
look @christopherthielen I left an `else if` in place instead of refactoring it to look nice